app-backup/amanda/amanda-3.5.1-r2.ebuild

```diff
@@ -75,13 +80,9 @@ amanda_variable_setup() {
 	currentamanda="$(set | egrep "^AMANDA_" | grep -v '^AMANDA_ENV_SETTINGS' | xargs)"
 
 	# First we set the defaults
-	[[ -z "${AMANDA_GROUP_GID}" ]] && AMANDA_GROUP_GID=87
 	[[ -z "${AMANDA_GROUP_NAME}" ]] && AMANDA_GROUP_NAME=amanda
 	[[ -z "${AMANDA_USER_NAME}" ]] && AMANDA_USER_NAME=amanda
-	[[ -z "${AMANDA_USER_UID}" ]] && AMANDA_USER_UID=87
-	[[ -z "${AMANDA_USER_SH}" ]] && AMANDA_USER_SH=/bin/bash
 	[[ -z "${AMANDA_USER_HOMEDIR}" ]] && AMANDA_USER_HOMEDIR=/var/spool/amanda
-	[[ -z "${AMANDA_USER_GROUPS}" ]] && AMANDA_USER_GROUPS="${AMANDA_GROUP_NAME}"
```

Comment:
can't you inline all of this? seeing that `AMANDA_USER_HOMEDIR` is general cant be changed from the ebuild anymore